docs/source/start-decoding.rst

```diff
@@ -87,11 +87,17 @@ The following table includes the csv parser operation.  This command is
 useful for when data messages are in a comma delimited format.  This command
 will instruct data in the csv to be mapped to either a sensor or date/time.
 
-+----------------------+----------------------------------------------------------+
-| **Command**          | **Description**                                          |
-+======================+==========================================================+
-| csv(sens#,...)       | Parse a series of comma separated values                 |
-+----------------------+----------------------------------------------------------+
++-----------------------------+------------------------------------------------------------+
+| **Command**                 | **Description**                                            |
++=============================+============================================================+
+| csv(sens#,...)              | Parse a series of comma separated values                   |
++-----------------------------+------------------------------------------------------------+
+| csv(delimiter=\|,sens#,...) | Parse a series of values separated by a specific delimiter |
+|                             | In this example the pipe, \|, character is used.           |
```

Comment:
remove commas and add either single or double quotes around the sample delimiter in the text like you do for the whitespace characters below.

Reply:
Took me a minute to figure out what was actually wrong but good call.